webpack.config.js

```diff
@@ -38,6 +38,9 @@ const webpackConfig = {
 		devtoolModuleFilenameTemplate: 'app:///[resource-path]'
 	},
 	module: {
+		// avoids this warning:
+		// https://github.com/localForage/localForage/issues/577
+		noParse: /node_modules\/localforage\/dist\/localforage.js/,
```

Comment:
That's an odd response from `localForage`. I wonder _why_ they are doing it this way.